tools/l10n/check_locales.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Check imported locales with release locales."""
+
+from os import path, listdir
+from sys import exit
+from locales import RELEASE_LOCALES
+
+PATH = path.join(path.dirname(path.abspath(__file__)), '../../app/src/main/res/')
+IGNORED_DIRECTORIES = ['sw600dp']
+
+def check_locales():
+    print("Checking for imported locales...")
+    got_error = False
+    imported_locales = [
+        directory[7:].replace('-r', '-') for directory in listdir(PATH) if directory.startswith('values-')
```

Comment:
Why `directory[7:]`? What are we removing (iiuc, we're getting the list from index 7 to the end)?

Indexing is a fragile way to remove these entries – is there a better way to get these locales?

Reply:
It gets list of directories starting with `values-` name, and then removes that `values-` part from their name. Locale directories are like `values-<locale>`. Usually `replace` is used, but we already filtered list before doing any operation, so it will always work.